test/wallet-http-test.js

```diff
@@ -483,18 +483,34 @@ describe('Wallet HTTP', function() {
     await assert.rejects(fn, {message: 'Lockup is required.'});
   });
 
-  it('should send bid with 0 value and 0 lockup', async () => {
+  it('should send bid with 0 value and non-dust lockup', async () => {
     await wallet.client.post(`/wallet/${wallet.id}/open`, {
       name: name
     });
 
     await mineBlocks(treeInterval + 1, cbAddress);
 
     await wallet.client.post(`/wallet/${wallet.id}/bid`, {
```

Comment:
The wallet http methods have been included in a `hs-client` release so we no longer have to use the underlying `client.post` method. Out of scope for this PR but low hanging fruit for another one.

Reply:
Good catch, opened: https://github.com/handshake-org/hsd/issues/486